### PR TITLE
Separate gitconfig sections with a newline

### PR DESCRIPTION
**Pull request by injust**
~~Edited by hand and didn't build to test.~~

### CODE REVIEW

**Comment by uncenter** [comment]
As I have just opened a Whiskers PR you should probably wait until it is merged before making these changes. The diff is annoying to resolve and if my pull request is merged first you will have to redo this anyway.

**Comment by injust** [comment]
Just wanted to make sure it's tracked somewhere, I can rebase once #8 is merged.

**Comment by uncenter** [comment]
When "rebasing" please make sure your changes to the template actually result in the changes you wrote by hand in the output files. You can do this by building the themes with Whiskers, installation instructions available at https://github.com/catppuccin/whiskers#installation.

**Comment by Anomalocaridid** [comment]
Like @uncenter said, could you please build the theme to test it after you rebase? That is a hard requirement for the PR being merged.

**Comment by injust** [comment]
Rebased and built. n.b. there's an extra trailing newline in `catppuccin.gitconfig` now.

**Comment by uncenter** [comment]
> there's an extra trailing newline in catppuccin.gitconfig now.

Pretty sure this is already the case?

**Comment by uncenter** [comment]
Oh I see what you mean. I have a fix, one sec.

File: delta.tera
Comment (uncenter) [inline delta.tera:42]:
```suggestion
{% if not loop.last %}
{% endif -%}
```
Make sure to build the theme after this to get rid of the extra trailing newline.

**Comment by injust** [comment]
Looks good now, thanks for the fix!

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
> Looks good now, thanks for the fix!

No problem. We had a similar issue with another port, will note it down to add that tip to the docs!

**Comment by Anomalocaridid** [comment]
Thank you!

**Pull request merged**
